File: cpp-examples/hello-sum/hello-sum.cpp
#include <iostream>
#include <stdlib.h>

int main( int argc, char *argv[] )
{
  std::cout << "Hello, Sum!\n";
  std::cout << "Literal: The sum of 2 and 3 is 5.\n";

  if (argc > 2) {
    std::cout << "Calculated: The sum of " << argv[1] << " and " << argv[2]
      << " is " << atoi(argv[1]) + atoi(argv[2]) << ".\n";
  }

  return 0;
}

File: python-examples/hello-sum/hello-sum.py
import sys

def main(argv):
    print('Hello, Sum!')
    print('The sum of 2 and 3 is 5.')
    sum = int(argv[1]) + int(argv[2])
    print('The sum of {0} and {1} is {2}.'.format(argv[1], argv[2], sum))

main(sys.argv)

